scripts/compile/common.sh

```diff
@@ -133,6 +136,7 @@ $CFLAGS_BUILD_TYPE \
 "
 
 COMMON_CXXFLAGS=" \
+-std=c++2b \
```

Comment:
```suggestion
-std=c++20 \
```

could we align the c++ version with react-native?

Reply:
Adding in the c++20 flags explicitly, I can see that WebKit itself is still sprinkling in  `-std=c++2b` in several places. I'll try and patch that out, and hope that it doesn't cause a cascade of new issues. If it does, then we'll need c++23 and NDK 28 to move forward with the Bun alignment.